socialite/src/components/StackOverflow/SearchResult.js

```diff
@@ -31,73 +34,110 @@ function SearchResult(props){
+                      <div className="tag px-3 py-2 mr-1 my-1">#{trending_tag['name']}</div>
+                      ))}
+                    </div>
+                  </div>
+                  <div className="row">
+                    <div className="col-xl-9">
+                      <div className="explore-posts">
+                        <div className="posts-list">
+                          {post_list && post_list.map(post => ( 
+                            <div className="single-post p-3 my-2" onClick={() => showIssue(post['id'])}>
+                              <div className="post">
+                                <div className="post-body">
+                                  <div className="post-header">{post['title']}</div>
+                                  <div className="post-content mt-4">{post ? Parser(post['body']) : ""}</div>
+                                </div>
+                              </div>
+                              <hr/>
+                              <div className="question-info mt-4">
+                                <div className="tags d-inline-block text-left">
+                                  {post.tags && Object.keys(post.tags).map(tag => ( 
+                                  <div className="tag px-3 py-2 mr-1 my-1">#{tag}</div>
+                                  ))}
+                                </div>
+                                <div className="info-details text-right">
+                                  <div className="small-upvote-count d-inline-block mr-2 px-2 py-1">{post['answers'].length} answers</div>
+                                </div>
+                              </div>
+                            </div>
+                          ))} 
                         </div>
-                        <div className="question-info mt-4">
-                          <div className="tags d-inline-block">
-                            {post.tags && Object.keys(post.tags).map(tag => ( 
-                            <div className="tag px-3 py-2 mr-1 my-1">#{tag}</div>
+                      </div>
+                    </div>
+                    <div className="col-xl-3 right-sidebar desktop-only">
+                      <div className="text-center">
+                        <button className="my-1 create-post w-100 py-3" onClick={createPost}>CREATE POST + </button>
+                      </div>
+                      <div className="my-3 p-2 top-users-post">
+                        <div className="card-header">
+                          TOP USERS
+                        </div>
+                        <div className="top-users m-4">
+                          <ul>
+                          {people_list && people_list.map(person => ( 
+                            <li className="mt-4">{person['username']}
+                              <div className="no-post float-right">{person['times_answered']} posts</div>
```

Comment:
It would be better if "posts" is changed to answers as we are calling questions as "posts".